.rubocop.yml

```diff
@@ -27,6 +27,15 @@ Documentation:
 Metrics/ClassLength:
   Max: 150
 
+Metrics/MethodLength:
+  Exclude:
+    - app/services/json_species.rb
+
+Metrics/AbcSize:
+  Exclude:
+    - app/services/json_species.rb
```

Comment:
We got app/services/json_species.rb:83:3: C: Assignment Branch Condition size for attributes is too high. [16/15]

'''ruby
def attributes
    def attributes
    {
      name: name,
      description: description,
      source_page: source_page,
      source_book: source_book,
      brawn: brawn,
      agility: agility,
      intellect: intellect,
      cunning: cunning,
      willpower: willpower,
      presence: presence,
      wound_threshold: wound_threshold,
      strain_threshold: strain_threshold,
      experience: experience,
      skill_name: skill_name,
      skill_start_rank: skill_start_rank,
      skill_limit_rank: skill_limit_rank
    }
  end
'''
Because we are making de assignation inside { }. I talked to Cejas yesterday to get another input on this matter and we couldn't figure out how to change this code. Therefore we decided to exclude this check on this file.

And the other one app/services/json_species.rb:83:3: C: Method has too many lines. [18/10]
was because of the same function.